src/maps/components/StandardMap.vue

```diff
@@ -160,6 +162,10 @@ export default {
       type: Object,
       default: null,
     },
+    paddingTopLeft: {
+      type: Array,
+      default: null,
```

Comment:
Why is default `null` but not `() => ([0, 0])`?

Reply:
I checked the leaflet library and they use `null` as default, so I switched to that everywhere now as default.